src/lib/format/icp_ledger.rs

```diff
@@ -0,0 +1,25 @@
+use candid::Decode;
+use icp_ledger::{Tokens, TransferError};
+
+use crate::lib::{e8s_to_tokens, AnyhowResult};
+
+pub fn display_transfer(blob: &[u8]) -> AnyhowResult<String> {
+    let result = Decode!(blob, Result<u64, TransferError>)?;
+    match result {
+        Ok(index) => Ok(format!("Transfer sent at block index {index}")),
+        Err(e) => Ok(format!("Transfer error: {e}")),
+    }
+}
+
+pub fn display_send_dfx(blob: &[u8]) -> AnyhowResult<String> {
+    let index = Decode!(blob, u64)?;
+    Ok(format!("Transfer sent at block index {index}"))
+}
+
+pub fn display_account_balance_or_dfx(blob: &[u8]) -> AnyhowResult<String> {
+    let tokens = Decode!(blob, Tokens)?;
+    Ok(format!(
+        "Balance: {}",
```

Comment:
Is there a reason why this doesn't include ICP as the unit?
```suggestion
        "Balance: {} ICP",
```